apps/schools/api_views/boundary.py

```diff
@@ -26,7 +26,7 @@ class AssemblyList(KLPListAPIView):
     bbox_filter_field = 'coord'
 
     def get_queryset(self):
-        qset = Assembly.objects.all()
+        qset = Assembly.objects.none()
 
```

Comment:
We don't have assembly data for Odisha yet. This reflects the Electoral boundaries at `http://odisha.ilp.org.in/reports/search`.